web/templates/navbar.hbs

```diff
@@ -17,7 +17,7 @@
                     </div>
                     <div id="searchbox">
                         <form id="searchbox_form" class="navbar-search">
-                            <div id="search_arrows" class="input-append">
+                            <div id="searchbox_container" class="input-append">
```

Comment:
I'm open to updating other names though I don't have great ideas for what to do with the names, since there's a lot of nesting. 

Updated to `searchbox-input-container`